src/scss/components/_content-title.scss

```diff
@@ -26,4 +26,10 @@
 
     .c-contentTitle-action {
         font-weight: $font-weight-bold;
+        text-decoration: none;
```

Comment:
Accessibility Q – is this action obvious enough for anyone who is colourblind if you get rid of the underline?

I know we've pushed back on things like this in the past (like on the [registration pages](https://www.just-eat.es/account/register) in the t&cs text, where it isn't obvious otherwise that it's a clickable link)

Reply:
V good point - I'll raise with Design.

Reply:
@ashleynolan  The context and positioning (right aligned to the title) and Medium weight of the fon (above screenshots are in the incorrect font) should help with this. The styling for this CTA link follows our default CTA link styling that's in use across the product. If there is a concern this styling is not accessible, it'd possibly need a full site review of some kind.

Reply:
@piet-je Yeh – it'd be good if we could look at this potentially in the future.  I'd be interested to see how it performed in accessibility tests with real users (not sure if that's something we do across our designs – I know we've done it in the past).